modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyQuery.jsx

```diff
@@ -266,9 +248,36 @@ function VocabularyQuery(props) {
           </MenuItem>
           );
       });
-      showUserEntry && suggestions.push(<Divider key="divider"/>);
     }
+
+    var allRequestsFailed = Object.keys(statuses).filter(vocab => !statuses[vocab]).length == 0;
+    var allRequestsSucceded = Object.keys(statuses).filter(vocab => statuses[vocab]).length == 0;
+
+    if (!allRequestsSucceded) {
+      suggestions.push(
+        <MenuItem
+          className={classes.dropdownMessage}
+          key="error-message"
+          disabled={true}
+        >
+          <Typography
+            component="p"
+            variant="caption"
+            color="error"
+          >
+            { allRequestsFailed && "Answer suggestions cannot be loaded for this question. Please inform your administrator." }
+            { !allRequestsFailed && !allRequestsSucceded && "Some answer suggestions for this question could not be loaded. Please inform your administrator." }
+          </Typography>
+        </MenuItem>
+      );
+      Object.keys(statuses).filter(vocab => statuses[vocab]).map( vocab => {
+          console.error("Cannot load answer suggestions from " + vocab);
+        }
+      );
+    }
+
     if (showUserEntry) {
+      suggestions.push(<Divider key="divider"/>);
```

Comment:
This divider should only be added only if there are any items above to separate this one from, otherwise it looks weird. Hopefully this works fine (not tested):
```suggestion
      suggestions.length > 0 && suggestions.push(<Divider key="divider"/>);
```

Reply:
Here is an example when we need a
![111111](https://user-images.githubusercontent.com/4371097/125003530-68b47200-e025-11eb-9e96-482d0b5b85f2.png)
 divider and suggestions.length == 0

Reply:
If we're displaying the message, doesn't that mean we added it to the suggestions array on line 260, which makes suggestions.length > 0 at this point?